### ISSUE TITLE
Live Streaming not working  in Chrome Beta 110

### ISSUE THREAD

**Issue by deepakbanwala0001** [muaz-khan/RTCMultiConnection#1061]
Hi,
Live Streaming is not working the Chrome Beta 110 (Release: January 12, 2023). If we have using the chrome latest version streaming not working. In Mozilla firefox, it's working fine.
Please find the link of muazkhan live-streaming demo in below:
https://muazkhan.com:9001/demos/video-conferencing.html

**Comment by deepakbanwala0001** [comment]
Chrome fixed 15 security vulnerabilities in the latest version. One security is related to WebRTC. 
Please find the reference blog of the WebRTC out-of-bounds read in WebRTC fixed in the latest chrome.

https://www.ghacks.net/2023/02/08/google-chrome-110-stable-fixes-15-security-vulnerabilities/

Google classifies three of the security issues with a severity rating of high, five as medium threats, and two as low threats. Among the vulnerabilities is a a type-confusion vulnerability in the JavaScript engine V8, an out-of-bounds read in WebRTC, and a use after free in GPU.


It may impact the live Streaming feature of the WebRTC.

**Comment by brittonk** [comment]
^ Yeah, my video conference Many to Many installation stopped working on Edge and Chrome after this week's browser updates. Currently only works in Firefox and Safari.

**Comment by deepakbanwala0001** [comment]
> ^ Yeah, my video conference Many to Many installation stopped working on Edge and Chrome after this week's browser updates. Currently only works in Firefox and Safari.

Any Solution?

**Comment by brittonk** [comment]
Yes-- through deep digging I found that in Chromium 110 the ICE servers will no longer allow query strings. So in the last ICE server from RTCMultichat, just delete it as "?transport=udp" will break the chat.

So... change:
`			connection.iceServers = [{
				'urls': [
					'stun:stun.l.google.com:19302',
					'stun:stun1.l.google.com:19302',
					'stun:stun2.l.google.com:19302',
					'stun:stun3.l.google.com:19302',
					'stun:stun4.l.google.com:19302',
					'stun:stun.l.google.com:19302?transport=udp',
				]
			}];`

To:
`			connection.iceServers = [{
				'urls': [
					'stun:stun.l.google.com:19302',
					'stun:stun1.l.google.com:19302',
					'stun:stun2.l.google.com:19302',
					'stun:stun3.l.google.com:19302',
					'stun:stun4.l.google.com:19302',
				]
			}];`

**Comment by SteveBamelis** [comment]
@brittonk , you are a lifesaver. This fixed it indeed.
